### DISCUSSION TITLE
Plot data inside GUI panels/widgets?

### DISCUSSION THREAD

**Discussion by barkov00** [emanuelhuber/RGPR#80] [Ideas]
Hi!
Im interested to create GUI for viewer (moving along X axis, processing options, fields with constants). 

Is It possible to put plot window inside Tcl/Tk interface?

**Comment by emanuelhuber** [comment]
Hello @barkov00 

Interesting question. Do you want to call R from tcl or to run tcl/tk from R? 
Check these links:

- https://wiki.tcl-lang.org/page/R
- https://www.stat.berkeley.edu/users/spector/s244/gui/Gui1.html

You should also have a look at Shiny (a web framework for building web applications using R).

Does it help?

Best,

Emanuel
